config/m4/module.m4

```diff
@@ -281,27 +273,48 @@ AC_DEFUN([_CHECK_MODULE_PACKAGE],
 #     - if set static lib 
 #       1. check it using package configuration (--static) and try to
 #          detect whole set of dependencies
-#       2. check it using includes/ldflags provided by user, if failed
+#       2. check it using includes/ldflags provided by user
 #     - for dynamic lib
 #       1. check for lib using declarations/link
 #       2. check for package
 #
 # Usage: UCX_CHECK_MODULE([id], [name], [prefix], [pkgname],
-#                         [[doc], [doc-lib], [doc-static or '-' to skip], [doc-failed]],
-#                         [action-if-found], [action-if-not-found],
-#                         [[headers], [decls], [includes], [libs - space separated], [funcs], [otherlibs]],
+#                         [[help], [help-lib], [help-static], [fail-msg]],
+#                         [if-found], [if-not-found],
+#                         [[headers], [decls], [includes], [libs], [funcs], [otherlibs]],
 #                         [alt-dirs])
 #
+# Parameters:
+#   id           - module ID, used to generate runtime configuration parameters --with-id
+#   name         - readable module name
+#   prefix       - variable prefix, in success variables prefix_CFLAGS and
+#                  prefix_LIBS will store module specific flags
```

Comment:
why id, name, and prefix can all be the same
(and prefix would be the capitalization of "id"/"name")?